src/lights/SpotLightShadow.js

```diff
@@ -20,6 +21,8 @@ SpotLightShadow.prototype = Object.assign( Object.create( LightShadow.prototype
 
 	update: function ( light ) {
 
+		if ( ! this.cameraAutoUpdate ) return;
```

Comment:
Instead, I would do the following in the calling method:
```javascript
if ( shadow.isSpotLightShadow && shadow.cameraAutoUpdate ) {

	shadow.update( light );

}
```

Reply:
done